util/util.go

```diff
@@ -166,29 +165,24 @@ func DecodeKey(k string) (*sym.Key, error) {
 
 // SetupDefaultLoggingConfig sets up a standard logging configuration.
 func SetupDefaultLoggingConfig(repoPath string) {
-	lj := &lumberjack.Logger{
-		Filename:   filepath.Join(repoPath, "log"),
-		MaxSize:    10, // megabytes
-		MaxBackups: 3,
-		MaxAge:     30, // days
-	}
-	backendFile := logger.NewLogBackend(lj, "", 0)
-	logger.SetBackend(backendFile)
-	logger.SetFormatter(logger.MustStringFormatter(logging.LogFormats["color"]))
-	logging.SetAllLoggers(logger.ERROR)
+	os.Setenv("GOLOG_LOG_FMT", "color")
+	os.Setenv("GOLOG_FILE", filepath.Join(repoPath, "log", "textile.log"))
+	logging.SetupLogging()
+	logging.SetAllLoggers(logging.LevelError)
 }
 
 // SetLogLevels sets levels for the given systems.
-func SetLogLevels(systems map[string]logger.Level) error {
+func SetLogLevels(systems map[string]logging.LogLevel) error {
 	for sys, level := range systems {
+		l := zapcore.Level(level)
```

Comment:
here `level` is a `logging.LogLevel` which is just an alias for `zapcore.Level`, but I can't call any `zapcore.Level` directly on it. I have to explicitly create a `zapcore.Level` from the `logging.LogLevel`. Expected behavior?